src/components/ActivityMap/AltitudeMap.tsx

```diff
@@ -22,6 +23,29 @@ const BRAND_ORANGE = '#FF4500';
 // this controls how many points are skipped before rendering another
 const SAMPLING_RATE = 5;
 
+function matchMeanElevation(data: AltitudePoint[]): AltitudePoint[] {
+  const stravaMean = mean(
+    data.flatMap(({ stravaAltitude }) =>
+      !!stravaAltitude ? stravaAltitude : []
+    )
+  );
+
+  const garminMean = mean(
+    data.flatMap(({ garminAltitude }) =>
+      !!garminAltitude ? garminAltitude : []
+    )
+  );
+
+  const meanDiff = stravaMean - garminMean || 0;
```

Comment:
Why the `|| 0`, how can this ever be falsey and not 0?

Reply:
if `isNan(garminMean)`, then the left operand will be falsy, and the correct diff is `0` (instead of the diff being `nan`)